core/builtin.py

```diff
@@ -540,11 +540,17 @@ def Dirs(argv, dir_stack):
   if arg.c:
     del dir_stack[:]
   elif arg.v:
-    for i, entry in enumerate(dir_stack):
-      print('%2d %s' % (i, entry))
+    temp_stack = dir_stack[:]
```

Comment:
How about unifying -v and -p under `PrintDirStack()` ?  It should just have a second parameter indicating what format to print it in.

Then the os.getcwd() and the sys.stdout.flush() can be shared among all 3 options.

Reply:
Sure, I think that can be done. I was keeping it separated for organizational reasons, but it would be more efficient to consolidate that code.

Reply:
I added a mode argument (integer) to the PrintDirStack() function but I'd rather not have magic numbers. At the moment, I just have the constants defined right before the function, like so:
```python
WITH_PREFIX = 1
WITHOUT_PREFIX =2
def PrintDirStack(dir_stack, mode):
  if mode == WITH_PREFIX:
    #etc etc etc
```
Is it fine to just define those constants right before the function or is there somewhere else I should be putting them?